src/bramble/client/main.js

```diff
@@ -588,6 +604,27 @@ define([
                 });
             }
 
+            function getChildrenOfFolder(folderPath, callback) {
+                var children = [];
+
+                function storeRelativePath(childPath, next) {
+                    // We don't care about child folders, only files
```

Comment:
Can you add that info to the comment please?  Keep no secrets when doing special cases.